README.md

```diff
@@ -30,3 +30,19 @@ docker-compose up
 ```
 
 and nwserver will spin up.
+
+## Install Builder
+
+You can ignore the `How to Launch` section up to `docker-compose up` by using `Builder`. To install is simply 
+adding this folder to your solution directory. From there you will need to add this snippet to your `.csproj` file.
+```
+  <Target Name="PostBuild" AfterTargets="PostBuildEvent">
```

Comment:
Do we want to add this to the actual template csproj too, or leave it as an option?

Reply:
Originally I had this in the csproj in a commit. When I edited the readme I realize maybe someone doesn't want that in there project file so I took it out and made note of it in the readme instead.